cms/search/signal_handlers.py

```diff
@@ -14,6 +14,10 @@
 logger = logging.getLogger(__name__)
 
 
+def _included_locales() -> list[str]:
```

Comment:
Nit: Maybe change func name to languages, as it is returning language codes not locales.